app/pb_kits/playbook/pb_label_pill/_label_pill.scss

```diff
@@ -0,0 +1,8 @@
+.pb_label_pill {
```

Comment:
You need to import any sass files from the kits you are using at the top. This will make sure your kit works by itself.

`@import "../pb_caption/caption";`
`@import "../pb_pill/pill";`